src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -1745,8 +1911,12 @@ void AutomationInstrumentClipView::auditionPadAction(int32_t velocity, int32_t y
 					instrumentClipView.sendAuditionNote(false, yDisplay, 64, 0);
 				}
 			}
+			display->cancelPopup();
```

Comment:
added back cancelPopup() which was removed by mistake in a previous PR. Without this it makes the audition pad popup remain on screen on 7SEG.